ambassador/tests/test_docker.py

```diff
@@ -51,6 +53,9 @@ def check_http(logfile) -> bool:
         return False
 
 def test_docker():
+    if not DockerImage:
+        pytest.fail('no DockerImage')
```

Comment:
This failure message will make me `grep`, which will lead me to what I need to do (set the env var). But can we make the message more explicit so I can skip the search and reading?

Reply:
There's actually a better error message below. 🙄  Removed this one.